feed-generator/Post.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace FeedGenerator
+{
+    using System;
+
+    internal class Post
```

Comment:
nit: `Post` doesn't seem good naming :)

Reply:
What about SocialMediaMessage?  What do you suggest?

Reply:
SocialMediaMessage is good.

Reply:
Renamed